packages/core/src/external/aws/lambda-logic/bundle-to-html-shared.ts

```diff
@@ -136,3 +136,17 @@ export function createSection(title: string, tableContents: string, id?: string)
     </div>
   `;
 }
+
+export const getDeceasedStatus = (familyMemberHistory: FamilyMemberHistory): string => {
+  const deceasedBoolean = familyMemberHistory.deceasedBoolean;
+  const conditionContributedToDeath = familyMemberHistory.condition?.find(condition => {
+    return condition.contributedToDeath;
+  });
+
+  if (deceasedBoolean !== undefined) {
+    return deceasedBoolean ? "yes" : "no";
+  } else if (conditionContributedToDeath?.contributedToDeath !== undefined) {
+    return conditionContributedToDeath?.contributedToDeath ? "yes" : "no";
+  }
+  return "";
```

Comment:
Cleanup:

```
function asYesNo(value: boolean): "yes" | "no" {
  return boolean ? "yes" as const : "no" as const;
}

export const getDeceasedStatus = (familyMemberHistory: FamilyMemberHistory): string => {
  const deceasedBoolean =  familyMemberHistory.deceasedBoolean
  if (deceasedBoolean !== undefined) {
    return asYesNo(deceasedBoolean);
  }
  
  const conditionContributedToDeath = familyMemberHistory.condition?.find(
    condition => condition.contributedToDeath
  );
  
  if (conditionContributedToDeath !== undefined) {
    return asYesNo(conditionContributedToDeath) 
  }
  
  return "";
};
```

Reply:
asYesNo seems a bit overkill but ill just add it for now

Reply:
yeah ur probably right there